ide/image/src/org/netbeans/modules/image/ImageDataObject.java

```diff
@@ -353,13 +333,8 @@ public ImageHeightProperty() {
 
           /** Gets value of property. Overrides superclass method. */
           public Integer getValue() throws InvocationTargetException {
```

Comment:
Is it intentional, that the `InvocationTargetException` was not removed? This is in contrast to new line 238.

Reply:
same as above